### PR TITLE
fix(apps/interface): fix color on trade pages

### PR DESCRIPTION
**Pull request by asaadam**
fix color on trade pages

refrence : https://www.figma.com/file/7Pzq1Gxxud84944o9kO2HM/Website-1.1?node-id=5%3A10525

### CODE REVIEW

File: apps/interface/components/SwapHistoryCard/index.tsx [hunk 1]
```diff
@@ -27,10 +27,9 @@ export const SwapHistoryCard = (props: SwapHistoryCardProps) => {
     // Styles
     const gray1 = useColorModeValue("gray.light.1", "gray.dark.1");
     const gray2 = useColorModeValue("gray.light.2", "gray.dark.2");
-    const gray4 = useColorModeValue("gray.light.4", "gray.dark.4");
     const gray10 = useColorModeValue("gray.light.10", "gray.dark.10");
     const gray12 = useColorModeValue("gray.light.12", "gray.dark.12");
-
+    const gray3 = useColorModeValue("gray.light.3", "gray.dark.3");
     return (
```
Comment (pyk) [inline apps/interface/components/SwapHistoryCard/index.tsx:33]: add whitespace here

File: apps/interface/components/SwapCard/index.tsx [hunk 2]
```diff
@@ -35,18 +35,17 @@ export const SwapCard = (props: SwapCardProps) => {
...
-    const gray2 = useColorModeValue("gray.light.2", "gray.dark.2");
     const gray3 = useColorModeValue("gray.light.3", "gray.dark.3");
     const gray10 = useColorModeValue("gray.light.10", "gray.dark.10");
     const gray12 = useColorModeValue("gray.light.12", "gray.dark.12");
 
     return (
         <Tabs
             data-testid="SwapCard"
-            background={gray1}
+            background={gray3}
```
Comment (pyk) [inline apps/interface/components/SwapCard/index.tsx:45]: This breaks the color on dark mode

File: apps/interface/components/SwapCard/index.tsx [hunk 3]
```diff
@@ -67,7 +66,7 @@ export const SwapCard = (props: SwapCardProps) => {
                         _selected={{
                             color: gray12,
                             fontWeight: "semibold",
-                            background: gray3,
+                            background: gray1,
```
Comment (pyk) [inline apps/interface/components/SwapCard/index.tsx:69]: This breaks the color on dark mode

File: apps/interface/components/SwapCard/index.tsx [hunk 4]
```diff
@@ -84,7 +83,7 @@ export const SwapCard = (props: SwapCardProps) => {
...
```
Comment (pyk) [inline apps/interface/components/SwapCard/index.tsx:86]: This breaks the color on dark mode

**Review by pyk — CHANGES_REQUESTED** [review]
This changes only fix the light mode but break the dark mode color

File: apps/interface/components/SwapCard/index.tsx (hunk 2 above)
Comment (asaadam) [reply apps/interface/components/SwapCard/index.tsx:45]: ups i forgot to check color on dark mode. 
on it ser

**Comment by pyk** [comment]
@asaadam This still break the darkmode

Screen Shot 2022-08-10 at 11 51 07
Screen Shot 2022-08-10 at 11 51 14


expected to be like this:

Screen Shot 2022-08-10 at 11 52 01
Screen Shot 2022-08-10 at 11 51 56

**Review by pyk — APPROVED** [review]
Thanks @asaadam  💯

**Pull request merged**